front/public/index.html

```diff
@@ -20,7 +19,9 @@
     <noscript>JavaScript is required to run this app.</noscript>
     <div id="app"></div>
 
-    <script type="text/javascript" src="/js/app.js"></script>
-    <script type="text/javascript" src="/js/linkage.js"></script>
+    <script type="text/javascript" src="/app.js"></script>
+    <script>
+        Elm.Main.init({node: document.getElementById("app")});
```

Comment:
Why place this JS within HTML? The noise when editing ports/etc. goes up notably (as well as extra indents).

Reply:
If this is strictly to avoid one additional req/resp, my inclination is to not hinder the source. We should be serving over http2, so the optimization is likely exceptionally minimal.

Reply:
@daved Let's add the additional file when that complexity is warranted. We don't have ports at the moment.